samples/DotNet/Azure.Messaging.EventHubs/EventHubsCaptureEventGridDemo/DWDumper/Program.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Avro.File;
+using Avro.Generic;
+using Newtonsoft.Json;
+using Microsoft.WindowsAzure.Storage;
```

Comment:
Move usings under namespace.

Reply:
I'd recommend going in the other direction.   

Looking over the set, the only one that has usings nested in the namespace is the anomaly detector.  My preference would be to see that elevate the using statements outside of the namespaces.    That said, so long as each project is intended to be self-contained and the convention is consistent at the project level, I don't have a strong opinion about changing them.